src/devtools/views/vuex/VuexStateInspector.vue

```diff
@@ -50,12 +54,39 @@ export default {
     return {
       showStateCopiedMessage: false,
       showBadJSONMessage: false,
-      showImportStatePopup: false
+      showImportStatePopup: false,
+      filter: '',
     }
   },
-  computed: mapGetters('vuex', [
-    'inspectedState'
-  ]),
+  computed: {
+    ...mapGetters('vuex', [
+      'inspectedState'
+    ]),
+    filteredState() {
+      const filtered = {};
```

Comment:
What do you think about reusing what is in ComponentStateInspector? So we just need to build the array of properties before filtering them:

```js
const inspectedState = [].concat(
        ...Object.keys(this.inspectedState).map(
          type => Object.keys(this.inspectedState[type]).map(
            key => ({
              key,
              type,
              value: this.inspectedState[type][key]
            })
          )
        )
      )

      return groupBy(sortByKey(inspectedState.filter(el => {
        return searchDeepInObject({
          [el.key]: el.value
        }, this.filter)
      })), 'type')
```

It would probably be nice to refactor the last part in the future so it's reused

Reply:
Good idea, I'll do it first thing when I have some spare time.

Reply:
Don't worry, I can add it myself as well. Thanks a lot for taking the time to contribute to the devtools! 😄